lib/bump/tasks.rb

```diff
@@ -14,8 +14,8 @@
       desc "Bump #{bump} part of gem version"
     end
 
-    task bump do
-      run_bump.call(bump, {})
+    task bump, :tag do |task, args|
+      run_bump.call(bump, {:tag => args[:tag]})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.